platformio.ini

```diff
@@ -24,7 +24,7 @@ build_flags = -fmax-errors=5
   -ggdb
   #-D USE_STDPERIPH_DRIVER
   #-D STM32F103xB
-  -D HSE_VALUE=14187200U
+  #-D HSE_VALUE=14187200U
```

Comment:
This breaks my serial, so disabled for debugging.

Reply:
Fair enough, still needs fixed